galasa-ui/src/components/headers/ThemeSelector.tsx

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright contributors to the Galasa project
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+'use client';
+
+import React, { useTransition } from "react";
+import { Dropdown } from "@carbon/react";
+import styles from "@/styles/Selector.module.css"; 
+import { useTheme } from "@/contexts/ThemeContext";
+import { useTranslations } from "next-intl";
+
+type ThemeType = "white" | "g100"; 
+
+const themes = [
+  { id: "white", text: "White", value: "white" },
+  { id: "g100", text: "Dark", value: "g100" },
```

Comment:
Instead of "White", the text should be "Light"

Also, looks like we're missing an option for setting the theme based on system preference?